### ISSUE TITLE
Missing BouncyCastle Dep

### ISSUE THREAD

**Issue by kferrone** [de-jcup/ekube#63]
When I try and connect with the credentials provided by k3s I get the following error:
```
An internal error occurred during: "connect to kubernetes - use context:pi-cluster".
JcaPEMKeyConverter is provided by BouncyCastle, an optional dependency. To use support for EC Keys you must explicitly add this dependency to classpath.
```
Can this simply be added as a Maven dep?

**Comment by de-jcup** [comment]
At https://github.com/fabric8io/kubernetes-client/issues/1796 there is a discussion about this.
They didn't add it to the maven plugin because it was a "heavy dependency".

"Adding that at a maven dep " - this is an eclipse plugin, so we could either 
- add an additional library eclipse plugin and provide the missing bouncy castle dependencies
- add bouncy castle as normal dependency inside the ekube plugin (simpler approach, so prefereable...)

So I would do following steps:
- Check version dependencies as in former link described and transform from maven to gradle:
```xml
 <dependency>
      <groupId>org.bouncycastle</groupId>
      <artifactId>bcprov-ext-jdk15on</artifactId>
      <version>${bouncycastle.version}</version>
    </dependency>
    <dependency>
      <groupId>org.bouncycastle</groupId>
      <artifactId>bcpkix-jdk15on</artifactId>
      <version>${bouncycastle.version}</version>
    </dependency>
  </dependencies>
```
This example references a bouncy castle version which is avaliable for JDK 1.5

https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on/1.68
https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk15on/1.68

==>
```groovy
dependencies {
       // https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on
      compile group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.68'

     // https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk15on
     compile group: 'org.bouncycastle', name: 'bcpkix-jdk15on', version: '1.68'
}
``` 

- bouncy castle dependencies should be added to gradle dependencies in 
  https://github.com/de-jcup/ekube/blob/master/ekube-other/build.gradle
- After this "./gradlew provideLibraries" must be called at root project folder in terminal
- A junit test should be added inside ekube-plugin/src/test/java so a check for JcaPEMKeyConverter should be done there to ensure bouncy castle dependency is available at compile time
 (just create https://www.bouncycastle.org/docs/pkixdocs1.5on/org/bouncycastle/openssl/jcajce/JcaPEMKeyConverter.html) 
- Test the plugin works in PDE environment
- Push to master / create PR
- Release...

**Comment by de-jcup** [comment]
@kferrone : I just released 0.9.0 on eclipse marketplace -  it contains the dependencies. So should work now

**Comment by kferrone** [comment]
It sure does!
